pkgs/tools/graphics/waifu2x-converter-cpp/default.nix

```
@@ -0,0 +1,34 @@
+{ cmake, fetchFromGitHub, opencv3, stdenv, opencl-headers
+, cudaSupport ? false, cudatoolkit ? null
+}:
+
+stdenv.mkDerivation rec {
+  name = "waifu2x-converter-cpp-${version}";
```

```suggestion
  pname = "waifu2x-converter-cpp";
```
Nix will do the rest on its own